### PR TITLE
French Translation

### PR DESCRIPTION
**Pull request by Firokat**
May not be perfect, but it's better than nothing no ?

### CODE REVIEW

File: locales/fr.lua [hunk 1]
```diff
@@ -0,0 +1,77 @@
+Locales['fr'] = {
+	-- Global
+	['custom_kick'] = 'INPUT CUSTOM KICK MESSAGE',
```
Comment (FreakOutModz) [inline locales/fr.lua:3]: This is not french

File: locales/fr.lua [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	['garage_no_aircrafts'] = 'Vous ne possedez pas de véhicules aériens',
+	['aircraft_is_impounded'] = 'Votre véhicule aérien n\'est pas ici',
+	-- Boats
+	['garage_boats'] = 'Garage de bateaux',
+	['pound_boats'] = 'Fourrière : Frais $%s',
+	['garage_no_boats'] = 'Vous ne possedez pas de bateaux',
+	['boat_is_impounded'] = 'Votre bateau n\'est pas ici',
+	-- Cars
+	['garage_cars'] = 'Garage',
+	['pound_cars'] = 'Fourrière: Fee $%s',
```
Comment (FreakOutModz) [inline locales/fr.lua:74]: This is not french

**Review by ghost — CHANGES_REQUESTED** [review]
This is a good attempt, but the chosen text uses bad French syntax, several typing errors. I do not recommend the merge of this.

File: locales/fr.lua (hunk 1 above)
Comment (Firokat) [reply locales/fr.lua:3]: This is not intended to be seen.

**Comment by Firokat** [comment]
In fact, as I said in the original description, it isn't perfect and the code is free to edit. Still, the translation is fairly good and can be used.

**Pull request closed without merging**